src/main/sensors/rpm_filter.c

```diff
@@ -61,17 +62,19 @@ static rpmNotchFilter_t filters[2];
 static rpmNotchFilter_t* gyroFilter;
 static rpmNotchFilter_t* dtermFilter;
 
-PG_REGISTER_WITH_RESET_FN(rpmFilterConfig_t, rpmFilterConfig, PG_RPM_FILTER_CONFIG, 2);
+PG_REGISTER_WITH_RESET_FN(rpmFilterConfig_t, rpmFilterConfig, PG_RPM_FILTER_CONFIG, 3);
```

Comment:
This does not strictly need an increment. Parameter groups can handle values being appended at the end without a reset, incrementing here will cause the configuration to be reset when updating the firmware. (https://github.com/betaflight/betaflight/blob/master/docs/development/ParameterGroups.md)